### PR TITLE
[AIRMOB-273] moved create wallet to home screen + tab bar icons fix

### PR DESCRIPTION
**Pull request by ilyaAir**

### CODE REVIEW

**Review by JavidHaji-zada — CHANGES_REQUESTED** [review]
- Inside CreateWalletStep0,  the `Header`  also gets `horizontal padding` from the container view, resulting in additional blank spaces.
- Create new wallet button should directly navigate to "CreateWalletStep0"

**Comment by ilyaAir** [comment]
image

File: src/screens/Wallet/CreateWallet/CreateWalletStep0.tsx [hunk 1]
```diff
@@ -26,99 +28,110 @@ export const CreateWalletStep0 = () => {
...
+              type="square"
+              onValueChange={setSelected}
+              value={selected}
+            />
+            <Spacer horizontal value={scale(12)} />
+            <Text
+              fontSize={16}
+              fontFamily="Inter_500Medium"
+              color={COLORS.nero}
+              style={{ maxWidth: scale(330) }}
```
Comment (ilyaAir) [inline src/screens/Wallet/CreateWallet/CreateWalletStep0.tsx:109]: @JavidHaji-zada wdyt if we leave it like that?

Comment (JavidHaji-zada) [reply src/screens/Wallet/CreateWallet/CreateWalletStep0.tsx:109]: I tried removing `maxWidth`, nothing changed. So, I think we can remove that

Comment (ilyaAir) [reply src/screens/Wallet/CreateWallet/CreateWalletStep0.tsx:109]: If i remove it, on iphone 14 pro, this text doesn't split to next line and it's weird

**Review by JavidHaji-zada — APPROVED** [review]

**Pull request merged**